### PR TITLE
Gantry Part II

### PR DESCRIPTION
**Pull request by sgtmind**
Adds in the gantry from BayStation12. Now with possible fixes

## About The Pull Request
New Human Provocateurs have been added. Three Vessels, and their corresponding Holopads and Ship Names might be seen.
## Why It's Good For The Game
Adds more variety to offsite roles, plus it adds a challenge
## Did You Test It?
I tested it and it seemed to work. 
## Authorship
Based from here: https://github.com/Baystation12/Baystation12/pull/29023
## Changelog

:cl:
/:cl:

### CODE REVIEW

**Comment by Anton-Kr** [comment]
Add this to away_sites_testing so that CI can run through it.

**Review by Anton-Kr — CHANGES_REQUESTED** [review]
Add CI to map.

**Comment by sgtmind** [comment]
How do I do that?

**Comment by sgtmind** [comment]
> Add CI to map.

Think I got that fixed

**Comment by Anton-Kr** [comment]
Add away site to maps/torch/torch.dm

File: maps/away/scavver/scavver_gantry.dm [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+	description = "Salvage Gantry turned Ship."
+	suffixes = list("scavver/scavver_gantry-1.dmm","scavver/scavver_gantry-2.dmm")
+	cost = 1
+	accessibility_weight = 10
+	shuttles_to_initialise = list(
+		/datum/shuttle/autodock/overmap/scavver_gantry,
+		/datum/shuttle/autodock/overmap/scavver_gantry/two,
+		/datum/shuttle/autodock/ferry/gantry
+	)
+   area_usage_test_exempted_root_areas = list(/area/scavver)
```
Comment (Anton-Kr) [inline maps/away/scavver/scavver_gantry.dm]: Still using spaces here. Using github to edit your PRs tend to cause this issue.

Comment (sgtmind) [reply maps/away/scavver/scavver_gantry.dm]: I used the standard DM

**Comment by sgtmind** [comment]
Anything else?

File: maps/away/scavver/scavver_gantry.dm [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+/datum/map_template/ruin/away_site/scavver_gantry
+	name = "Salvage Gantry"
+	id = "awaysite_gantry"
+	description = "Salvage Gantry turned Ship."
+	suffixes = list("scavver/scavver_gantry-1.dmm","scavver/scavver_gantry-2.dmm")
+	cost = 1
+	accessibility_weight = 10
+	area_usage_test_exempted_root_areas = list(/area/scavver)
+	apc_test_exempt_areas = list(
+        /area/scavver = NO_SCRUBBER|NO_VENT|NO_APC
```
Comment (Anton-Kr) [inline maps/away/scavver/scavver_gantry.dm]: bad indent

File: maps/away/scavver/scavver_gantry.dm [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    )
```
Comment (Anton-Kr) [inline maps/away/scavver/scavver_gantry.dm]: bad indent

File: maps/away/scavver/scavver_gantry.dm [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+		"nav_gantry_four",
+		"nav_gantry_five"
+	)
+	initial_restricted_waypoints = list(
+		"ITV The Reclaimer" = list("nav_hangar_gantry_one"),
+		"ITV Vulcan" = list("nav_hangar_gantry_two")
+	)
+
+/obj/item/mech_component/sensors/light/salvage/prebuild()
+  ..()
```
Comment (Anton-Kr) [inline maps/away/scavver/scavver_gantry.dm]: bad indent

File: maps/away/scavver/scavver_gantry.dm [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+  software.installed_software = list(MECH_SOFTWARE_UTILITY, MECH_SOFTWARE_ENGINEERING)
```
Comment (Anton-Kr) [inline maps/away/scavver/scavver_gantry.dm]: bad indent

**Review by Anton-Kr — CHANGES_REQUESTED** [review]

File: maps/away/scavver/scavver_gantry.dm [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+/datum/map_template/ruin/away_site/scavver_gantry
+	name = "Salvage Gantry"
+	id = "awaysite_gantry"
+	description = "Salvage Gantry turned Ship."
+	suffixes = list("scavver/scavver_gantry-1.dmm","scavver/scavver_gantry-2.dmm")
+	cost = 1
+	accessibility_weight = 10
+	area_usage_test_exempted_root_areas = list(/area/scavver)
+	apc_test_exempt_areas = list(
```
Comment (Anton-Kr) [inline maps/away/scavver/scavver_gantry.dm:12]: This should only include areas that actually do not have these things. For instance if area/scavver/foo has no APC, it should be shown as such /area/scavver/foo = NO_APC. This is causing CI failures.

**Review by Anton-Kr — CHANGES_REQUESTED** [review]
Further there are a few pipes that have the wrong directions.

**Pull request closed without merging**